src/vsd/shared/utils/get_objectstore_file.py

```diff
@@ -36,5 +36,5 @@ def get_objectstore_file(location, dir1, connection):
     parser.add_argument("filename")
     parser.add_argument("-s", "--swift_connection")
     args = parser.parse_args()
-    tmpdir = os.getenv("TMPDIR", "/tmp/")
+    tmpdir = os.getenv("SHARED_DIR", "/tmp/")
```

Comment:
Not in the scope of this PR, but I was wondering how we deal with environment variables. Here we use the `os` module. Most other places use `environs.Env`. We should probably use one way.

With type annotations in mind I'd like to suggest not using `os` or even `environs` but using Pydantic's settings management instead: https://pydantic-docs.helpmanual.io/usage/settings/

Reply:
This one is worth discussion with everyone.

Reply:
Agreed. Already posted something on Slack